captum/insights/frontend/src/components/Contributions.tsx

```diff
@@ -1,9 +1,14 @@
 import React from "react";
 import styles from "../App.module.css";
 import { calcHSLFromScore } from "../utils/color";
+import { FeatureOutput } from "../models/visualizationOutput";
 
-function Contributions(props) {
-  return props.feature_outputs.map((f) => {
+interface ContributionsProps {
+  feature_outputs: FeatureOutput[];
+}
+
+function Contributions(props: ContributionsProps) {
+  return <>{props.feature_outputs.map((f) => {
```

Comment:
out of curiosity, why does this need fragments?

Reply:
`props.feature_outputs.map` returns array of component which gives type error as react component return should be either a `literal` type or single `jsx` element.